WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift

```diff
@@ -1,6 +1,7 @@
 import Foundation
 import SwiftUI
 import WordPressFlux
+import SwiftUI
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/104b0c52a1168c8bc73da5fee16d942972ba7c48/WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift#L4"></span>Imports should be unique<br />
<code>duplicate_imports</code> <code>SiteSettingsViewController+Swift.swift:4</code></td>
    </tr>
  </tbody>
</table>